src/pagination/pagination.ts

```diff
@@ -224,4 +199,40 @@ export class NgbPagination implements OnChanges {
       this.pageChange.emit(this.page);
     }
   }
+
+  private _updatePages(newPage: number) {
+    // re-calculate new length of pages, should always have at least 1
```

Comment:
I think you always have at least one page in pagination, otherwise it would be only `<` and `>` arrows, which looks strange